src/test/java/gov/healthit/chpl/aqa/pageObjects/SearchPage.java

```diff
@@ -210,4 +210,12 @@ public static WebElement loadChplID(final WebDriver driver, final String chplID)
     public static WebElement certStatusFilterOptions(final WebDriver driver) {
         return driver.findElement(By.cssSelector("#filters > div.col-sm-8 > div.btn-group.dropdown.open > ul"));
     }
+
+    public static WebElement surveillanceDropdown(final WebDriver driver) {
+        return driver.findElement(By.xpath("//button[@id='surveillance-toggle']"));
+    }
+
+    public static WebElement complaintsReporting(final WebDriver driver) {
+        return driver.findElement(By.xpath("//*[@id=\"surveillance-dropdown-menu\"]/li[4]/a"));
```

Comment:
Specifying the fourth element is brittle. Would instead rather have you have the method return a list of the nav links that are in this dropdown, and then loop through them to see if the title appears

Reply:
Now, I'm using a different page object instead of writing a method.

Reply:
This is now looking for any link that has the title "Complaints Reporting", right? I think that's too broad. There ought to be some xpath / selector / something that gets to `surveillance menu > has the right title`, or something like that